libs/langchain-community/src/document_loaders/web/cheerio.ts

```diff
@@ -7,36 +7,21 @@ import type {
 import { Document } from "@langchain/core/documents";
 import {
   AsyncCaller,
-  AsyncCallerParams,
 } from "@langchain/core/utils/async_caller";
 import { BaseDocumentLoader } from "@langchain/core/document_loaders/base";
-import type { DocumentLoader } from "@langchain/core/document_loaders/base";
+import { WebBaseLoaderParams, WebBaseLoader } from "./web_base_loader.js";
 
 /**
  * Represents the parameters for configuring the CheerioWebBaseLoader. It
- * extends the AsyncCallerParams interface and adds additional parameters
- * specific to web-based loaders.
+ * extends the WebBaseLoaderParams interface and adds additional parameters
+ * specific to loading with Cheerio.
  */
-export interface WebBaseLoaderParams extends AsyncCallerParams {
```

Comment:
Actually one small thing - this is exported, so on the off-chance someone is importing and using it, would prefer to keep the old export and deprecate here.

Reply:
Ah yep, fair enough. Will sort that out now. :+1: